packs/basin/east.json

```diff
@@ -46,11 +46,11 @@
         },
         "maps/jungle-city/east.json/entities/215/settings/npcStates/1/event/trade/event/5/message": {
                 "orig": "Only need certain items for it.",
-                "text": "Besoin juste de quelques items pour ça."
+                "text": "Besoin juste de quelques objets pour ça."
         },
         "maps/jungle-city/east.json/entities/215/settings/npcStates/1/event/trade/event/6/traded/1/message": {
                 "orig": "Been a while since I held this in my hands. ",
-                "text": "Ça fait un bail depuis la dernière fois que j'ai tenu ceci dans mes mains."
+                "text": "Ça fait un bail que j'ai ce truc dans mes mains."
```

Comment:
Quand je traduis, j'ai à la fois la version anglaise et allemande sous les yeux (les devs sont allemands).
Et la version allemande est assez claire: "Ist eine Weile her, dass ich den das letzte Mal in der Hand hatte."  Même si je l'ai traduit un peu littéralement...

Reply:
Peut-être "Cela fait un moment que je ne l'ai pas tenu dans mes mains" alors ? Je trouve que c'est difficile de mettre "Ça fait un bail" avec quelque chose d'autres après.